File: frontend/src/content/site.ts
// Dile bagli OLMAYAN sabitler. Metinler locales/*.json'da durur.

export const SITE = {
  name: 'Vårt Marked',
  domain: 'https://vartmarked.no',
  phone: '+4792249005',
  phoneDisplay: '92 24 90 05',
  whatsapp: '4792249005',
  email: 'post@vartmarked.no',
  orgNr: '915 026 915',
  founded: 2016,
  address: {
    street: 'Fossumveien 60',
    postalCode: '0985',
    city: 'Oslo',
    country: 'NO',
    lat: 59.962428,
    lng: 10.9224773,
  },
  social: {
    facebook: 'https://www.facebook.com/vaartmarked',
    instagram: 'https://www.instagram.com/vaartmarked/',
    tiktok: 'https://www.tiktok.com/@vaartmarked',
  },
  mapEmbed:
    'https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d1998.5!2d10.9224773!3d59.962428!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x46417aa38ec3b99f%3A0xdeda1579832c1ed3!2sV%C3%A5rt%20Marked!5e0!3m2!1sno!2sno!4v1',
} as const

export const STATS = [
  { key: 'yearsLabel', value: '25+' },
  { key: 'itemsLabel', value: '3000+' },
  { key: 'daysLabel', value: '7' },
] as const

export const FEATURES = [
  { key: 'fresh', icon: '🥦' },
  { key: 'international', icon: '🌍' },
  { key: 'prices', icon: '💰' },
  { key: 'experience', icon: '🤝' },
] as const

export const PRODUCTS = [
  { key: 'vegetables', icon: '🥬' },
  { key: 'fruit', icon: '🍎' },
  { key: 'meat', icon: '🥩' },
  { key: 'fish', icon: '🐟' },
  { key: 'dairy', icon: '🧀' },
  { key: 'bakery', icon: '🍞' },
  { key: 'specialties', icon: '🫒' },
] as const

// Acilis saatleri: gun anahtari + saat. Saatler dilden bagimsiz.
export const HOURS = [
  { key: 'mon', opens: '08:00', closes: '21:00', schemaDay: 'Monday' },
  { key: 'tue', opens: '08:00', closes: '21:00', schemaDay: 'Tuesday' },
  { key: 'wed', opens: '08:00', closes: '21:00', schemaDay: 'Wednesday' },
  { key: 'thu', opens: '08:00', closes: '21:00', schemaDay: 'Thursday' },
  { key: 'fri', opens: '08:00', closes: '21:00', schemaDay: 'Friday' },
  { key: 'sat', opens: '08:00', closes: '21:00', schemaDay: 'Saturday' },
  { key: 'sun', opens: '10:00', closes: '19:00', schemaDay: 'Sunday' },
] as const

// Marka adlari cevrilmez; yalniz iki tanesi dile bagli (contact.payment.*)
export const PAYMENT_BRANDS = ['Visa', 'Mastercard', 'Amex'] as const
export const PAYMENT_TRANSLATED = ['contactless', 'cash'] as const

File: backend/VartMarked.Prerender/Site.cs
namespace VartMarked.Prerender;

/// <summary>Dile bagli olmayan sabitler - frontend'deki src/content/site.ts ile ayni degerler.</summary>
public static class Site
{
    public const string Name = "Vårt Marked";
    public const string Domain = "https://vartmarked.no";
    public const string Phone = "+4792249005";
    public const string Email = "post@vartmarked.no";
    public const string OrgNr = "915 026 915";
    public const string Street = "Fossumveien 60";
    public const string PostalCode = "0988";
    public const string City = "Oslo";
    public const string Country = "NO";
    public const double Lat = 59.962428;
    public const double Lng = 10.9224773;

    public static readonly string[] SameAs =
    [
        "https://www.facebook.com/vaartmarked",
        "https://www.instagram.com/vaartmarked/",
        "https://www.tiktok.com/@vaartmarked",
    ];

    public static readonly (string Day, string Opens, string Closes)[] Hours =
    [
        ("Monday", "08:00", "21:00"),
        ("Tuesday", "08:00", "21:00"),
        ("Wednesday", "08:00", "21:00"),
        ("Thursday", "08:00", "21:00"),
        ("Friday", "08:00", "21:00"),
        ("Saturday", "08:00", "21:00"),
        ("Sunday", "10:00", "19:00"),
    ];

    /// <summary>Hizmet verilen mahalleler - Groruddalen. Yerel aramada esleme gucunu artirir.</summary>
    public static readonly string[] AreaServed =
    [
        "Stovner", "Fossum", "Vestli", "Rommen", "Romsås", "Haugenstua", "Groruddalen", "Oslo",
    ];

    /// <summary>SSS anahtarlari - frontend'deki FAQ_KEYS ile ayni sirada olmali.</summary>
    public static readonly string[] FaqKeys = ["hvor", "apent", "halal", "land", "bestille"];
}
